catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/utils/FormulaEditorWrapper.java

```diff
@@ -109,6 +115,24 @@ public FormulaEditorWrapper checkShows(String expected) {
 		return new FormulaEditorWrapper();
 	}
 
+	public FormulaEditorWrapper checkHasCursorPosition(int expected) {
+		FormulaEditorEditText editText = (FormulaEditorEditText) getCurrentActivity()
```

Comment:
just saw this now. getCurrentActivity is not safe to be used like this. this is what custom matchers are intended for, so you dont have to aquire the activity like this.